flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/basics/StreamWindowSQLExample.java

```diff
@@ -87,8 +87,8 @@ public static void main(String[] args) throws Exception {
         env.execute("Streaming Window SQL Job");
 
         // should output:
-        // 2019-12-12 00:00:00.000,3,10,3
-        // 2019-12-12 00:00:05.000,3,6,2
+        // +I[2019-12-12 00:00:05.000, 3, 6, 2]
+        // +I[2019-12-12 00:00:00.000, 3, 10, 3]
```

Comment:
It seems that the output order is reversed. Is there any way to enforce the output order?

Reply:
@tsreaper - I've tried it several times and the order of the output is not maintained.  Please let me know if you have any recommendation.  Thanks.

Reply:
@tsreaper - I've changed the parallelism to 1 and not the output is in the same order as you had mentioned.  I've also added table output recommended by @JingsongLi.
Please let me know if you have any other recommendations.  Thanks

Reply:
I suppose using one of them is enough. The one used by the original example is using the PrintSink which may have multiple instances in a job and may produce out of order results (unless the user explicitly sets the parallelism to 1), the other one mentioned by Jingsong is using the CollectSinkFunction and it's parallelism is always 1 (no matter what the parallelism is in other parts of that job), thus there is no out of order issue.